README.md

```diff
@@ -286,6 +286,20 @@ This function is mostly used internally, and almost never should be called by yo
 
 jsdom does this for `Window`, which is written in custom, non-webidl2js-generated code, but inherits from `EventTarget`, which is generated by webidl2js.
 
+### For callback interfaces
+
+#### `convert(value, { context })`
+
+Performs the Web IDL conversion algorithm for this callback interface, converting _value_ into a function that performs [call a user object’s operation](https://heycam.github.io/webidl/#call-a-user-objects-operation) when called.
```

Comment:
What's the signature of the returned function? In particular, is there a parameter for setting the `this` value or is the `this` value of the returned function used?